public/app/features/storage/ExportPage.tsx

```diff
@@ -0,0 +1,263 @@
+            <h3>Status</h3>
+            <pre>{JSON.stringify(status, null, 2)}</pre>
+            {status.running && (
+              <div>
+                <Button variant="secondary" onClick={doStop}>
+                  Stop
+                </Button>
+              </div>
+            )}
+          </div>
+        )}
+
+        {!Boolean(status?.running) && (
+          <div>
+            <Field label="Format">
+              <Select
+                options={formats}
+                width={40}
+                value={formats.find((v) => v.value === body?.format)}
+                onChange={(v) => setBody({ ...body!, format: v.value! })}
+              />
+            </Field>
+            {body?.format === 'entityStore' && !config.featureToggles.entityStore && (
+              <div>
+                <Alert title="Missing feature flag">Enable the `entityStore` feature flag</Alert>
+              </div>
+            )}
+            {body?.format === 'git' && (
+              <>
+                <Field label="Keep history">
+                  <Switch
+                    value={body?.history}
+                    onChange={(v) => setBody({ ...body!, history: v.currentTarget.checked })}
+                  />
+                </Field>
+
+                <Field label="Include">
+                  <>
+                    <InlineFieldRow>
+                      <InlineField label="Toggle all" labelWidth={labelWith}>
+                        <InlineSwitch
+                          value={Object.keys(body?.exclude ?? {}).length === 0}
+                          onChange={(v) => setInclude('*', v.currentTarget.checked)}
+                        />
+                      </InlineField>
+                    </InlineFieldRow>
+                    {serverOptions.value && (
+                      <div>
+                        {serverOptions.value.exporters.map((ex) => (
+                          <InlineFieldRow key={ex.key}>
+                            <InlineField label={ex.name} labelWidth={labelWith} tooltip={ex.description}>
+                              <InlineSwitch
+                                value={body?.exclude?.[ex.key] !== true}
+                                onChange={(v) => setInclude(ex.key, v.currentTarget.checked)}
+                              />
+                            </InlineField>
+                          </InlineFieldRow>
+                        ))}
+                      </div>
+                    )}
+                  </>
+                </Field>
+              </>
+            )}
+
+            <Field label="General folder" description="Set the folder name for items without a real folder">
+              <Input
+                width={40}
+                value={body?.generalFolderPath ?? ''}
+                onChange={(v) => setBody({ ...body!, generalFolderPath: v.currentTarget.value })}
+                placeholder="root folder path"
+              />
+            </Field>
+
+            <HorizontalGroup>
```

Comment:
If on entitystore option and feature flag isn't enabled maybe we should also hide the rest of the section below? I can push up a commit in a little bit to address this 👍
<img width="1187" alt="Screenshot 2022-12-13 at 10 34 15 AM" src="https://user-images.githubusercontent.com/22381771/207416978-4dd9c4cc-9a04-4fcc-9069-42615fa7f29f.png">